### PR TITLE
Use pytest instead of setup.py test

### LINKED ISSUE #120: Use py.test instead of python setup.py test

### PR DESCRIPTION
**Pull request by asottile**
Resolves #120

### CODE REVIEW

**Comment by asottile** [comment]
\o/ passed travis and appveyor -> ready for review

**Comment by dahlia** [comment]
It would be better if CONTRIBUTING.rst is also updated.

**Comment by asottile** [comment]
CONTRIBUTING.rst doesn't seem to mention setup.py test, but that does remind me that tox needs updating

**Comment by asottile** [comment]
updated tox.ini

File: .travis.yml [hunk 1]
```diff
@@ -12,9 +12,12 @@ python:
 - 3.4
 - 3.5
 install:
-- pip install -e . "flake8 >= 2.4.0" "Werkzeug >= 0.9" coverage coveralls
+- pip install -rrequirements-dev.txt coveralls
 script:
-- coverage run --source sass,sassc,sassutils setup.py test
+- coverage run --source sass,sassc,sassutils -m pytest sasstests.py
```
Comment (dahlia) [inline .travis.yml]: How about using [pytest-cov][1]?

[1]: https://github.com/pytest-dev/pytest-cov

Comment (asottile) [inline .travis.yml]: could, but yagni?

Comment (dahlia) [inline .travis.yml]: Not necessary! :grin:

Comment (asottile) [inline .travis.yml]: I've never used it before, maybe we could add it in a followup.

**Pull request merged**